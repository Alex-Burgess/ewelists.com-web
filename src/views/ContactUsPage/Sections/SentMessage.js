import React from 'react';
// nodejs library to set properties for components
import PropTypes from "prop-types";
// @material-ui/core components
import { makeStyles } from "@material-ui/core/styles";

import styles from "assets/jss/custom/views/contactUsCustomStyle.js";
const useStyles = makeStyles(styles);

export default function SentMessage(props) {
  const classes = useStyles();
  const { name } = props;

  return (
    <div className={classes.sent}>
      <p>
        Thank you for your message {name}.  We will get back to you by email.
      </p>
    </div>
  )
}

SentMessage.propTypes = {
  name: PropTypes.string.isRequired
};
