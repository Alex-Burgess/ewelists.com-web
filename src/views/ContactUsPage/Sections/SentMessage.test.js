import * as React from 'react';
import TestRenderer from 'react-test-renderer';
import SentMessage from './SentMessage';

describe('Tests for SentMessage section.', () => {
  test('Renders message for name', () => {
    const tree = TestRenderer
      .create(<SentMessage name="Test User"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
