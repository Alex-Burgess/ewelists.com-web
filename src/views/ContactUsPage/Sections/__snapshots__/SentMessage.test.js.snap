// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for SentMessage section. Renders message for name 1`] = `
<div
  className="makeStyles-sent-20"
>
  <p>
    Thank you for your message 
    Test User
    .  We will get back to you by email.
  </p>
</div>
`;
